src/console2.sol

```diff
@@ -1,6 +1,10 @@
 // SPDX-License-Identifier: MIT
 pragma solidity >=0.4.22 <0.9.0;
 
+struct Console {
+    string _color;
+}
```

Comment:
Wonder if we could also add support for bold/underline/italics? I think @brockelmore often uses escape codes in his console.logs so curious to get his thoughts on this PR too

Reply:
Yeah definitely, wonder on how should I approach this in terms of syntax?

I can think of a few but would like to hear your opinions:

- console.red("bold").log(); -> If no font weight is specified it will use normal.
- console.red().bold().log(); -> Again, if bold() is not called it will just default to normal.
- console.color("red", "bold").log(); -> I personally prefer the ones above but, could be another option.